languages/kotlin/publish-local.sh

```diff
@@ -0,0 +1,11 @@
+mkdir -p ./sdk/src/main/jniLibs/{arm64-v8a,armeabi-v7a,x86_64,x86}
```

Comment:
```suggestion
cd "$(dirname "$0")"

mkdir -p ./sdk/src/main/jniLibs/{arm64-v8a,armeabi-v7a,x86_64,x86}
```
Can we add this to the top of the file to make sure the script can be run from any directory?

I tend to run all the scripts from the root sdk dir like `./languages/kotlin/publish-local.sh` if I can, and this would make it more convenient.